Models/Cars/CarFactory.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Models.Images;
+
+namespace Models.Cars
+{
+    public class CarFactory
+    {
+        private static CarFactory instance = null;
+
+        public static CarFactory GetInstance()
+        {
+            if (instance == null)
+            {
+                instance = new CarFactory();
+            }
+            return instance;
+        }
+
+        private CarFactory()
+        {
+        }
+
+        public CapturedCar CreateCapturedCar(Car car, Image image)
+        {
+            return new CapturedCar
+            {
+                Id = car.Id,
+                UserId = car.UserId,
+                LicensePlate = car.LicensePlate,
+                ColorName = car.ColorName,
+                MakeName = car.MakeName,
+                Model = car.Model,
+                BodyType = car.BodyType,
+                Year = car.Year,
+                Status = car.Status,
+                IsReported = car.IsReported,
+                Image = ImageFactory.GetInstance().CreateImage(image)
+            };
+        }
+
+        public Car CreateCar(int id, int userId, string licensePlate, string colorName,
+                            string makeName, string model, string bodyType, string year,
+                            CarStatus status, bool isReported)
+        {
+            return new Car
```

Comment:
Car and capturedCar seems almost identical, maybe make image an optional parameter and use 1 class, another solution would be inheritance